src/Config.hs

```diff
@@ -597,6 +603,14 @@ optionsParser = Options
         <> help ("Vault address, the scheme, either http:// or https://, the ip-address or DNS name, " ++
             "followed by the port, separated with a ':'." ++
             " Cannot be combined with either VAULT_PORT or VAULT_HOST")
+    authBackend
+      = maybeStrOption
+      $ long "auth-backend"
+      <> metavar "AUTH_BACKEND"
+      <> value Nothing
+      <> help ("Name of Vault authentication backend."
+      ++ "Defaults to 'kubernetes' or 'github' depending on the type of the"
```

Comment:
The lack of spaces at the end of these strings cause the thing that @ig11987 pointed out in the readme.